packages/pieces/community/google-sheets/package.json

```diff
@@ -1,4 +1,4 @@
 {
   "name": "@activepieces/piece-google-sheets",
-  "version": "0.10.3"
+  "version": "0.11.0"
```

Comment:
Since I was adding a new functionality I thought I had to update the minor version but either is fine for me. 

Just for my information then, the pieces version numbers aren't following Semantic Versioning ? If the minor number is used to indicate a breaking feature, the patch version number is therefore both for new feature and bugfixes ? When do you use the major version number ? (assuming you call the numbers major.minor.patch for version number)